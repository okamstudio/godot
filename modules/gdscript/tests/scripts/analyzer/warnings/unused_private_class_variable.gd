# GH-72135

var _a
@warning_ignore("unused_protected_class_variable")
var _b
@warning_ignore("unused_protected_class_variable") var _c
var _d

var __e
@warning_ignore("unused_private_class_variable")
var __f
@warning_ignore("unused_private_class_variable") var __g
var __h

func test():
	pass
