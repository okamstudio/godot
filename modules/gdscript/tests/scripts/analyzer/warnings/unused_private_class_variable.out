GDTEST_OK
~~ WARNING at line 3: (UNUSED_PROTECTED_CLASS_VARIABLE) The class variable "_a" is declared but never used in the class.
~~ WARNING at line 7: (UNUSED_PROTECTED_CLASS_VARIABLE) The class variable "_d" is declared but never used in the class.
~~ WARNING at line 9: (UNUSED_PRIVATE_CLASS_VARIABLE) The class variable "__e" is declared but never used in the class.
~~ WARNING at line 13: (UNUSED_PRIVATE_CLASS_VARIABLE) The class variable "__h" is declared but never used in the class.
